js-core/homeworks/homework-19/task-0.5/server.js

```diff
@@ -0,0 +1,25 @@
+const http = require('http');
+const fs = require('fs');
+const port = 3000;
+
+const requestHandler = (request, response) => {
+    let path = request.url;
+
+	if(path == '/'){
+		path = '/index.html';
+    }
+    
+    let index = fs.readFileSync(`.${path}`);
```

Comment:
the server will crash on custom URL for example https://localhost:3000/yes

because there is no file exists in file system